test/test-udp-try-send.c

```diff
@@ -104,14 +106,22 @@ TEST_IMPL(udp_try_send) {
   r = uv_udp_try_send(&client, &buf, 1, (const struct sockaddr*) &addr);
   ASSERT(r == UV_EMSGSIZE);
 
+  buf = uv_buf_init(buffer, sizeof(buffer));
+  r = uv_udp_try_send_ex(&client, &buf, 1, (const struct sockaddr*) &addr, INET_ADDRSTRLEN);
+  ASSERT(r == UV_EMSGSIZE);
+
   buf = uv_buf_init("EXIT", 4);
   r = uv_udp_try_send(&client, &buf, 1, (const struct sockaddr*) &addr);
   ASSERT(r == 4);
 
+  buf = uv_buf_init("EXIT", 4);
+  r = uv_udp_try_send_ex(&client, &buf, 1, (const struct sockaddr*) &addr, INET_ADDRSTRLEN);
```

Comment:
`INET_ADDRSTRLEN` is wrong (on other places as well), as it's about max. length of *textual* representation.  You want `sizeof(struct sockaddr_in)` in _this_ case.